optimize/components/userguide/combined-process-reports.md

```diff
@@ -1,63 +0,0 @@
----
-id: combined-process-reports
-title: Combined process reports
-description: Occasionally, it is necessary to compare multiple reports or visualize them together in one diagram.
----
-
-<span class="badge badge--platform">Camunda 7 only</span>
-
-Occasionally, it is necessary to compare multiple reports or visualize them together in one diagram. This can be achieved by creating a special type of report called a combined process report. To create a new combined process report, visit the **Collections** page and click **Create New > New Report > Combined Process Report**.
```

Comment:
There is a page in "Additional Features" called "Footer" - https://preview.docs.camunda.cloud/pr-4785/optimize/next/components/userguide/additional-features/footer/

Let's delete that as well, it's not very helpful and is quite C7-centric anyway